src/shared/components/featureTitle/FeatureTitle.spec.tsx

```diff
@@ -1,10 +1,9 @@
-import * as FeatureTitle from './FeatureTitle';
 import React from 'react';
 import { assert } from 'chai';
 import { shallow, mount } from 'enzyme';
 import sinon from 'sinon';
 
-describe('FeatureTitle', () => {
+describe('', () => {
```

Comment:
Actually, for our stats, we should try to keep files, even if we didn't write tests. However, we need to actually import the code for it to work (we don't here).  We can look into other ways to get untested code to register.